internal/account/account.go

```diff
@@ -13,11 +14,6 @@ type Account struct {
 	CreatedAt    time.Time `json:"created_at" db:"created_at"`
 }
 
-type AccountCredentialSpec struct {
-	Email    string `json:"email"`
-	Password string `json:"password"`
-}
```

Comment:
Can you elaborate why did u remove this and created LoginContract and RegisterContract instead, while knowing the content is exactly the same?

Reply:
I was thinking of moving the entire `specs` from all verticals to `contract.go`, I think that would help with separation of concern between logic focused `account.go` and request handling payload `contract.go`

Wdyt?

Reply:
Okay, I think it's ur idea is fine